src/patternfly/components/AlertGroup/alert-group.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../patternfly-utilities";
+
+// Tabs
+.pf-c-alert-group {
+  // Alert Group variables
+  --pf-c-alert-group__item--MarginBottom: var(--pf-global--spacer--sm);
+
+  // Toast variables
+  --pf-c-alert-group--m-toast--Top: 0;
+  --pf-c-alert-group--m-toast--Right: 0;
+  --pf-c-alert-group--m-toast--MaxWidth: #{pf-size-prem(600px)};
+  --pf-c-alert-group--m-toast--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-alert-group--m-toast--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-alert-group--m-toast--PaddingBottom: 0;
+  --pf-c-alert-group--m-toast--PaddingLeft: var(--pf-global--spacer--xl);
+
+  width: 100%;
+
+  // Spacing between alerts
+  > .pf-c-alert:not(:last-child) {
```

Comment:
It's lower specificity and less code to use `> * + *` (the [owl selector](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls), with `margin-top`.

Reply:
Oh that's very cool, thank you for sharing that! Would using `margin-top` in this case be an acceptable violation of [our component guideline](https://pf-next.com/guidelines#components)?

> Elements inside a component never use top margins. The first element touches the top of its component.

Reply:
Ah, nope, it's just referring to the first element, it should touch the top of the component.

Reply:
Great, I've made the change.